### PR TITLE
Preparation ih

### PR DESCRIPTION
**Pull request by eliegaboriau**
This PR updates the CI/CD and admin creation tasks.
It also removes a unused task

### CODE REVIEW

**Review by armandfardeau — APPROVED** [review]

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -0,0 +1,163 @@
+name: "CI/CD"
+on: [push]
+
+env:
+  CI: "true"
+  SIMPLECOV: "true"
+  RSPEC_FORMAT: "documentation"
+  RUBY_VERSION: 2.7.5
```
Comment (Quentinchampenois) [inline .github/workflows/ci_cd.yml:8]: CI's RUBY_VERSION does not match `.ruby-version`

File: Dockerfile [hunk 2]
```diff
@@ -1,36 +1,37 @@
-FROM ruby:2.7.1
+FROM ruby:2.7.5
```
Comment (Quentinchampenois) [inline Dockerfile:1]: Same here, Docker ruby version does not match application .ruby-version

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
We probably should bump application ruby-version from `2.7.1` to `2.7.5`

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
You must run `bundle install` to update the Gemfile.lock with the new ruby version

**Pull request merged**